README.md

```diff
@@ -21,6 +21,7 @@ meant to be able to change with it.
 
   * [Layout](#layout)
   * [Example Structure](#example-structure)
+  * [Example Group Structure](#example-group-structure)
```

Comment:
It makes sense to be before example structure

Reply:
Makes sense. Moved the section up.